src/main/spec/internal-subscriptions-sync-api-spec.yaml

```diff
@@ -134,13 +134,12 @@ paths:
       tags:
         - internalSubscriptions
       responses:
-        '202':
-          description: "The request for syncing organization's subscription is processing."
+        '200':
```

Comment:
This actually returns a 200 even though it was listed as a 202 here.

Reply:
Thanks!